Taskfile.Shared.yml

```
@@ -62,56 +62,31 @@ tasks:
     label: "{{.TASK}}-{{.NAME}}"
     dir: coprocessors/{{.NAME}}
     cmds:
-      - docker run -d --rm --name {{.NAME}}-co --env PORT="{{.COPROCESSOR_PORT}}" -m 1g --cpus=1 -p {{.COPROCESSOR_PORT}}:{{.COPROCESSOR_PORT}} {{.NAME}}-co
+      - docker run -d --rm --name coprocessor --net=task --env PORT="8000" -p 8000:8000 -m 1g --cpus=1 {{.NAME}}-co
     vars:
       NAME: "{{.NAME|default .CLI_ARGS}}"
   cleanup-coprocessor:
     label: "{{.TASK}}-{{.NAME}}"
     cmds:
-      - docker stop {{.NAME}}-co
+      - docker stop coprocessor
     vars:
       NAME: "{{.NAME|default .CLI_ARGS}}"
 
   start-subgraphs:
+    dir: subgraphs/fixed
     cmds:
-      - task: build-subgraph
-        vars:
-          NAME: a
-      - task: build-subgraph
-        vars:
-          NAME: b
-      - task: start-subgraph
-        vars:
-          NAME: a
-          PORT: 8081
-      - task: start-subgraph
-        vars:
-          NAME: b
-          PORT: 8080
+      - DOCKER_BUILDKIT=1 docker build . -t fixed
+      - docker run -d --name subgraph --net=task -m 1g --cpus=1 -p 8082:8082 fixed
+
   cleanup-subgraphs:
     cmds:
-      - task: cleanup-subgraph
-        vars:
-          NAME: a
-      - task: cleanup-subgraph
-        vars:
-          NAME: b
+      - docker stop subgraph
+      - docker rm subgraph
 
-  build-subgraph:
-    internal: true
-    label: "build-subgraph-{{.NAME}}"
-    dir: subgraphs/subgraph-{{.NAME}}
+  create-network:
     cmds:
-      - docker build . -t sub-{{.NAME}}
-    sources:
-      - "*.*"
-  start-subgraph:
-    internal: true
-    label: "start-subgraph-{{.NAME}}"
-    cmds:
-      - docker run --rm -d --name sub-{{.NAME}} --env PORT="{{.PORT}}" -m 1g --cpus=1 -p {{.PORT}}:{{.PORT}} sub-{{.NAME}}
-  cleanup-subgraph:
-    internal: true
-    label: "cleanup-subgraph-{{.NAME}}"
+      - docker network create task
+
+  remove-network:
     cmds:
-      - docker stop sub-{{.NAME}}
+      - docker network rm task
```

```suggestion
      - docker network rm task
  cleanup-fixed-subgraph:
    cmds:
      - docker stop fixed
```

the above is referenced below, and will error if not defined